collector/collector_test.go

```diff
@@ -0,0 +1,42 @@
+package collector
+
+import (
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+	dto "github.com/prometheus/client_model/go"
+)
+
+type LabelMap map[string]string
```

Comment:
Btw. (not really about this refactoring, so maybe do it later), there's https://github.com/prometheus/common/blob/master/model/labelset.go#L28 already. Even in case you don't want to import that package just for that type, maybe call it `LabelSet` instead of `LabelMap` to be consistent.